web/courses.go

```diff
@@ -349,6 +263,92 @@ func (s *AutograderService) isEmptyRepo(ctx context.Context, sc scm.SCM, request
 	return isEmpty(ctx, sc, repos)
 }
 
+// rejectEnrollment rejects a student enrollment, if a student repo exists for the given course, removes it from the SCM and database
+func (s *AutograderService) rejectEnrollment(ctx context.Context, sc scm.SCM, enrolled *pb.Enrollment) error {
+	// course and user are both preloaded, no need to query the database
+	course, user := enrolled.GetCourse(), enrolled.GetUser()
+	repos, err := s.db.GetRepositories(&pb.Repository{UserID: user.GetID(), OrganizationID: course.GetOrganizationID(), RepoType: pb.Repository_USER})
```

Comment:
Should use line breaks for the `pb.Repository{}` query instance.

Reply:
Fixed.